pkg/cmd/workspace/code.go

```diff
@@ -211,6 +224,20 @@ func openBrowserIDE(activeProfile config.Profile, workspaceId string, projectNam
 		}
 	}
 }
+// check if the port is ready to serve
+
+func checkPortReady(port uint16) bool {
```

Comment:
No need to define this function here. You can use the `ports.IsPortAvailable` function that is already available in our codebase.

Reply:
`IsPortAvailable` determines whether a given port is available for listening, while `CheckPortReady` checks if a service is ready on the specified port by making an HTTP GET request.

Reply:
`IsPortAvailable` falls short for checking service readiness.

Reply:
@harkiratsm you're right. An http GET check is better in this case.